client/api_client.js

```diff
@@ -81,6 +81,24 @@ _.extend(ApiClient.prototype, {
     return restClient(options);
   },
 
+  getPdf: function(path) {
+    var options = {
+      method: 'GET',
+      uri: this.urlBase + path,
+      json: true,
+      resolveWithFullResponse: true,
+      encoding: null,
+      headers: {
+        'Authorization': 'Bearer ' + createToken('GET', path, this.apiKeyId, this.serviceId),
+        'User-agent': 'NOTIFY-API-NODE-CLIENT/' + version,
+        "Content-type": "application/pdf"
```

Comment:
these headers are for the request. Content-type's only used on a request object when you're actually sending a body (such as a POST).

as such, I don't think we need this getPdf function at all, and can just use the regular `get` request in `getPdfForLetterNotification`

Reply:
Ok, I switched to use get request and it works if I add ` encoding: null` to the request options, is that ok?